### PR TITLE
Ciara Wheeler foodLog page

### PR DESCRIPTION
**Pull request by ciarawheeler**
1. basic design on the page and it's components
2. created an Add Food form (functionality is not there yet)
3. All of the places that need to navigate somewhere will navigate to ApiScreen currently

issue #43

### CODE REVIEW

File: frontend/app/screens/foodlog/FoodLog.tsx [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+export default function FoodLogScreen() {
+    // const calendarBar = (
+    //     <View style = {styles.calendarNav}>
+    //         <Ionicons name={"chevron-back-outline"} size={24} style={styles.leftArrow}></Ionicons>
+    //         <Ionicons name={"calendar-clear-outline"} size={24}></Ionicons>
+    //         {/* This will be "today" when it is the current date, if not it will display the date of the data they are viewing*/}
+    //         <Text>Today</Text>
+    //         <Ionicons name={"chevron-forward-outline"} size={24} style={styles.rightArrow}></Ionicons>
+    //     </View>
+    // );
```
Comment (sjiang1) [inline frontend/app/screens/foodlog/FoodLog.tsx:18]: Why do you want to keep this commented calendar bar?

**Comment by esnowgol** [comment]
Looks good! Similar to Michael's page, I'd recommend some input validation if you weren't already planning to. In your case, probably just prevent the user from using "-" in food calories/fat/carbs/protein. Food can't have negative calories, and no need to support subtraction "10-5 calories"

**Review by esnowgol — APPROVED** [review]
Looks Good.

Makes some more api requests than necessary, we can work on reducing that next sprint, which should allow us to speed it up.

Mine had some issues creating the day unless touching the calendar we can also address

Additionally, some loading indicators would be a nice touch.

**Comment by sjiang1** [comment]
Quality of Code: Good
Participation in Code Review: Good
Post-Review Changes: Good

**Pull request merged**